debian/postrm

```diff
@@ -37,13 +33,26 @@ case "$1" in
       deb-systemd-helper unmask jellyfin.service >/dev/null
     fi
 
+    # Remove user and group
     userdel jellyfin > /dev/null 2>&1 || true
     delgroup --quiet jellyfin > /dev/null 2>&1 || true
+    # Remove config dir
+    if [[ -d $CONFIGDATA ]]; then
+      rm -rf $CONFIGDATA
+    fi
+    # Remove log dir
+    if [[ -d $LOGDATA ]]; then
+      rm -rf $LOGDATA
+    fi
+    # Remove program data dir
     if [[ -d $PROGRAMDATA ]]; then
       rm -rf $PROGRAMDATA
     fi
+    # Remove binary symlink
     [[ -f /usr/bin/jellyfin ]] && rm /usr/bin/jellyfin
+    # Remove sudoers config
     [[ -f /etc/sudoers.d/jellyfin-sudoers ]] && rm /etc/sudoers.d/jellyfin-sudoers
+    # Remove /var/lib/jellyfin, just in case
     [[ -d /var/lib/jellyfin ]] && rm -rf /var/lib/jellyfin
```

Comment:
It will be with defaults, but if the user moves the data dir it might not be. I should actually change it to `-e` though, because we'd want to remove *anything* at that location (like a symlink, which is the sort of situation I was thinking of).

Reply:
Added entries with `-e` for each of the 3 default directories and a comment explaining the "why".